tests/Database/DatabaseEloquentIntegrationTests.php

```diff
@@ -82,6 +90,22 @@ public function testBasicModelRetrieval()
 	}
 
 
+	public function testBasicModelCollectionRetrieval()
+	{
+		EloquentTestUser::create(['id' => 1, 'email' => 'taylorotwell@gmail.com']);
+		EloquentTestUser::create(['id' => 2, 'email' => 'abigailotwell@gmail.com']);
+
+		$models = EloquentTestUser::oldest('id');
+
+		$this->assertEquals(2, $models->count());
+		$this->assertInstanceOf('Illuminate\Database\Eloquent\Collection', $models);
+		$this->assertInstanceOf('EloquentTestUser', $models[0]);
+		$this->assertInstanceOf('EloquentTestUser', $models[1]);
+		$this->assertInstanceOf('taylorotwell@gmail.com', $models[0]->email);
```

Comment:
It seems that these tests never ran, because it has the wrong suffix.

Here's the fix: https://github.com/laravel/framework/pull/7096

Thanks @JoostK